lib/array_contacts_repository.rb

```diff
@@ -0,0 +1,26 @@
+class ArrayContactsRepository 
```

Comment:
I would call this `InMemoryContactsRepository`. That's more indicative of its purpose - i.e. used when you don't want to go to the DB.